vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/DatabaseBackedProcessor.java

```diff
@@ -190,13 +190,13 @@ static String computeWorkflowRunHashId(
       Iterable<String> labels,
       TreeSet<String>
           inputIds, // In both existing calls, hashFromAnalysisId has already been called. TODO:
-                    // maybe refactor if any more calls are needed
+      // maybe refactor if any more calls are needed
       Collection<? extends ExternalId> externalIds) {
     try {
       final var digest = MessageDigest.getInstance("SHA-256");
       digest.update(name.getBytes(StandardCharsets.UTF_8));
       for (final var id : inputIds) {
-        final var idBytes = id.getBytes(StandardCharsets.UTF_8);
+        final var idBytes = hashFromAnalysisId(id).getBytes(StandardCharsets.UTF_8);
```

Comment:
hashFromAnalysisId has already been called, as line 192 describes.

Reply:
Oh I see; I had missed that the comment in the arguments referred to this line further down the method.

When you were doing your refactoring, did you run across a reason why we wouldn't want to change the two existing computeWorkflowRunHashId calls to pass in the full Vidarr identifier, so as to run hashFromAnalysisId inside that method only?